sdk/core/System.Net.ClientModel/README.md

```diff
@@ -0,0 +1,41 @@
+# System.Net.ClientModel shared client library for .NET
+
+`System.Net.ClientModel` provides shared primitives, abstractions, and helpers for .NET service client libraries.
+
+`System.Net.ClientModel` allows client libraries built from its components to expose common functionality in a consistent fashion, so that once you learn how to use these APIs in one client library, you'll know how to use them in other client libraries as well.
+
+## Getting started
+
+Typically, you will not need to install `System.Net.ClientModel`.
+it will be installed for you when you install one of the client libraries using it.
+
+### Install the package
+
+Install the client library for .NET with [NuGet](https://www.nuget.org/).
+
+### Prerequisites
+
+None needed for `System.Net.ClientModel`.
```

Comment:
This was a "quick and dirty" for the demo preview package.  We should update it to say something similar to what's in the Azure.Core README.